eth2deposit/intl/el/cli/generate_keys.json

```
@@ -0,0 +1,35 @@
+{
+    "validate_password": {
+        "msg_password_prompt": "Πληκτρολογήστε τον κωδικό πρόσβασης που ασφαλίζει τον χώρο αποθήκευσης κλειδιών του επαληθευτής σας",
+        "msg_password_confirm": "Επανάληψη για επιβεβαίωση",
+        "err_password_mismatch": "Σφάλμα: οι δύο εισαχθείσες τιμές δεν ταιριάζουν. Παρακαλώ πληκτρολογήστε ξανά."
+    },
+    "generate_keys_arguments_decorator": {
+        "num_validators": {
+            "argument": "--num_validators",
+            "help": "Ο αριθμός των κλειδιών επαληθευτών που θέλετε να δημιουργήσετε (μπορείτε πάντα να δημιουργήσετε περισσότερα αργότερα)",
+            "prompt": "Παρακαλώ επιλέξτε πόσους επαληθευτές θέλετε να εκτελέσετε"
+        },
+        "folder": {
+            "argument": "--folder",
+            "help": "Ο αριθμός των κλειδιών επαληθευτών που θέλετε να δημιουργήσετε (μπορείτε πάντα να δημιουργήσετε περισσότερα αργότερα)"
+        },
+        "chain": {
+            "argument": "--chain",
+            "help": "Η έκδοση του eth2 στην οποία στοχεύετε. Χρησιμοποιήστε το \"κεντρικό δίκτυο\" αν κάνετε κατάθεση ETH",
```

I'm not confident with my EL. But it since `mainnet` was translated here.
 
```suggestion
            "argument": "--chain",
            "help": "Η έκδοση του eth2 στην οποία στοχεύετε. Χρησιμοποιήστε το \"mainnet\" αν κάνετε κατάθεση ETH",
```